tests/wpt/mozilla/tests/mozilla/bluetooth/getCharacteristics/disconnect-invalidates-objects-with-uuid.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/_mozilla/mozilla/bluetooth/bluetooth-helpers.js"></script>
+<script>
+'use strict';
+promise_test(t => {
+    window.testRunner.setBluetoothMockDataSet(adapter_type.heart_rate);
+    return window.navigator.bluetooth.requestDevice({
+        filters: [{services: [heart_rate.name]}]
+    })
+    .then(device => device.gatt.connect())
+    .then(gattServer => {
+        return gattServer.getPrimaryService(heart_rate.name)
+        .then(service => service.getCharacteristics(body_sensor_location.name))
+        .then(c => {
+            let characteristics = [].concat(c);
```

Comment:
This is for ensure that the `characteristics` variable is an iterable object (in this case an Array), by default `c` is not iterable for some reason. Without this, we can't use the [let for ... of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) syntax in [line27](https://github.com/szeged/servo/pull/89/files/8f0cf985b3a8b6dfd93a09013a04d8a1a7865a10..fac70aca4a206a326a52918e0c02048854ab7ef0#diff-884d56cd987682247c62a5262484dad3R27), which needs characteristics to be iterable.